src/ui/organisms/Eip7702Table.tsx

```diff
@@ -0,0 +1,329 @@
+        </a>
+      </div>
+    );
+  };
+
+  return createCell;
+}
+
+interface WalletTypeCell {
+  typeString: string;
+  hasSmartWallet: boolean;
+}
+
+type CellValue = string | WalletTypeCell;
+
+export default function Eip7702Table(): React.JSX.Element {
+  const columnHelper = createColumnHelper<TableRow>();
+  const rankColumn = columnHelper.display({
+    id: 'rank',
+    header: '#',
+    cell: ({ row }): React.ReactNode => row.index + 1,
+  });
+  const walletNameColumn = columnHelper.accessor(
+    (row: TableRow): CellValue => row.wallet.metadata.displayName,
+    {
+      id: 'wallet',
+      header: 'Wallet',
+      cell: createWalletNameCell(),
+    },
+  );
+  const walletTypeColumn = columnHelper.display({
+    id: 'type',
+    header: 'Type',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      switch (info.row.original.typeFor7702) {
+        case WalletTypeFor7702.EIP7702:
+          return (
+            <div className='mt-1 flex flex-wrap gap-1'>
+              <EipStandardTag standard={eip7702} usePrefix={true} />
+            </div>
+          );
+        case WalletTypeFor7702.NON_7702_SMART_WALLET:
+          return (
+            <div className='mt-1 flex flex-wrap gap-1'>
+              <EipStandardTag standard={erc4337} usePrefix={true} />
+            </div>
+          );
+        case WalletTypeFor7702.NON_7702_EOA:
+          return <span>Non-7702 EOA</span>;
+        case WalletTypeFor7702.OTHER:
+          return <span>Non-7702</span>;
+      }
+    },
+  });
+  const walletContractColumn = columnHelper.display({
+    id: 'contract',
+    header: 'Contract',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      const contract = info.row.original.contract;
+
+      if (contract === 'UNKNOWN') {
+        return <span style={{ opacity: 1 }}>Unknown - Send PR!</span>;
+      }
+
+      return (
+        <ExternalLink
+          url={`https://etherscan.io/address/${contract.address}`}
+          defaultLabel={contract.name}
+        />
+      );
+    },
+  });
+  const walletContractSource = columnHelper.display({
+    id: 'contract-source',
+    header: 'Source',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      const contract = info.row.original.contract;
+
+      if (contract === 'UNKNOWN') {
+        return null;
+      }
+
+      if (!contract.sourceCode.available) {
+        return <span style={{ color: 'var(--color-error)' }}>Unavailable</span>;
+      }
+
+      const sourceRefs = refs(contract.sourceCode);
+      const sourceUrl = isNonEmptyArray(sourceRefs)
+        ? nonEmptyGet(nonEmptyGet(sourceRefs).urls)
+        : `https://etherscan.io/address/${contract.address}#code`;
```

Comment:
here you reuse the etherscan URL, so i tihnk as per previous remark that it would be great to move it in a single variable / const so we can reuse it more effetively.